packages/wdio-browserstack-service/src/util.ts

```diff
@@ -638,14 +638,140 @@ export function getCiInfo () {
         }
     }
     // Visual Studio Team Services
-    if (env.TF_BUILD === 'True') {
+    if (isTrue(env.TF_BUILD)) {
         return {
             name: 'Visual Studio Team Services',
             build_url: `${env.SYSTEM_TEAMFOUNDATIONSERVERURI}${env.SYSTEM_TEAMPROJECTID}`,
             job_name: env.SYSTEM_DEFINITIONID,
             build_number: env.BUILD_BUILDID
         }
     }
+    // Appveyor
+    if (isTrue(env.APPVEYOR)) {
+        return {
+            name: 'Appveyor',
+            build_url: `${env.APPVEYOR_URL}/project/${env.APPVEYOR_ACCOUNT_NAME}/${env.APPVEYOR_PROJECT_SLUG}/builds/${env.APPVEYOR_BUILD_ID}`,
+            job_name: env.APPVEYOR_JOB_NAME,
+            build_number: env.APPVEYOR_BUILD_NUMBER
+        }
+    }
+    // Azure CI
+    if (env.AZURE_HTTP_USER_AGENT && env.TF_BUILD) {
+        return {
+            name: 'Azure CI',
+            build_url: `${env.SYSTEM_TEAMFOUNDATIONSERVERURI}${env.SYSTEM_TEAMPROJECT}/_build/results?buildId=${env.BUILD_BUILDID}`,
+            job_name: env.BUILD_BUILDID,
+            build_number: env.BUILD_BUILDID
+        }
+    }
+    // AWS CodeBuild
+    if (env.CODEBUILD_BUILD_ID || env.CODEBUILD_RESOLVED_SOURCE_VERSION || env.CODEBUILD_SOURCE_VERSION) {
+        return {
+            name: 'AWS CodeBuild',
+            build_url: env.CODEBUILD_PUBLIC_BUILD_URL,
+            job_name: env.CODEBUILD_BUILD_ID,
+            build_number: env.CODEBUILD_BUILD_ID
+        }
+    }
+    // Bamboo
+    if (env.bamboo_buildNumber) {
+        return {
+            name: 'Bamboo',
+            build_url: env.bamboo_buildResultsUrl,
+            job_name: env.bamboo_shortJobName,
+            build_number: env.bamboo_buildNumber
+        }
+    }
+    // Wercker
+    if (env.WERCKER || env.WERCKER_MAIN_PIPELINE_STARTED) {
+        return {
+            name: 'Wercker',
+            build_url: env.WERCKER_BUILD_URL,
+            job_name: env.WERCKER_MAIN_PIPELINE_STARTED ? 'Main Pipeline' : null,
+            build_number: env.WERCKER_GIT_COMMIT
+        }
+    }
+    // Google Cloud
+    if (env.GCP_PROJECT || env.GCLOUD_PROJECT || env.GOOGLE_CLOUD_PROJECT) {
+        return {
+            name: 'Google Cloud',
+            build_url: null,
```

Comment:
Why are the url's set to null?

Reply:
Left it null in cases where that info is not available